src/components/DetailsBanner.test.jsx

```diff
@@ -18,6 +18,8 @@ it('Displays the banner image based on component name prop', () => {
   render(<DetailsBanner image={image} />);
 
   // from testing-library doc https://testing-library.com/docs/queries/byalttext/
-  const bannerImage = screen.getByAltText(image.alt); // pass string in to test
-  expect(bannerImage.src).toBe(image.src);
+  const bannerImage = screen.getAllByAltText(image.alt); // pass string in to test
+  expect(bannerImage[0].src).toBe(image.src);
+  expect(bannerImage[1].src).toBe(image.src);
+  expect(bannerImage[2].src).toBe(image.src);
```

Comment:
If you wanted to, you could add some tests for DetailsBanner handling your new `isWideImage` function.